superstruct/src/types.ts

```diff
@@ -8,10 +7,10 @@ import { validate, Struct } from 'superstruct';
 
 type Options = Parameters<typeof validate>[2];
 
-export type Resolver = <T extends Struct<any, any>>(
+export type Resolver = <T extends Struct<any, any>, TFieldValues, TContext>(
   schema: T,
   factoryOtions?: Options,
```

Comment:
```suggestion
  factoryOptions?: Options,
```

just because it was slightly bothering me 😄